CONTRIBUTING.md

```diff
@@ -6,16 +6,23 @@ Before interacting with the Redwood community, please read and understand our [C
 
 **Table of Contents**
 
-- [Contributing](#Contributing)
-  - [Local Development](#Local-Development)
-    - [Copy and Watch](#Copy-and-Watch)
-      - [Specifying a RW_PATH](#Specifying-a-RW_PATH)
-    - [Local Package Registry Emulation](#Local-Package-Registry-Emulation)
-      - [Setting Up and Running a Local NPM Registry](#Setting-Up-and-Running-a-Local-NPM-Registry)
-      - [Publishing a Package](#Publishing-a-Package)
-      - [Installing Published Packages in Your Redwood App](#Installing-Published-Packages-in-Your-Redwood-App)
-  - [Running Your Redwood App's Local Server(s)](#Running-Your-Redwood-App's-Local-Server(s))
-  - [Releases](#Releases)
+- [Contributing](#contributing)
```

Comment:
Can we eliminate this top-level in the TOC? Feels redundant, especially since it's _below_ the section it's linking.

Reply:
Totally.